docs/source/quantization.rst

```diff
@@ -39,7 +39,106 @@ APIs are provided that incorporate typical workflows of converting FP32 model
 to lower precision with minimal accuracy loss.
 
 Quantization API Summary
------------------------------
+------------------------
```

Comment:
nit: is this new code intended to be in the PR?

Reply:
no